Formula/secman.rb

```diff
@@ -0,0 +1,29 @@
+class Secman < Formula
+  desc "👊 Human-friendly and amazing secrets manager"
+  homepage "https://secman.dev/docs/cli"
+  url "https://cli-files.secman.dev/secman-v6.0.71/secman-v6.0.71.tar.gz"
+  sha256 "2a9c03df9b444e9c110b640c2e22ef4db147623cb728f7b2c68f2bc488d0e230"
+  license "MIT"
+
+  livecheck do
+    url "https://github.com/scmn-dev/secman/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  def install
+    inreplace "bin/secman", /^CLIENT_HOME=/, "export SECMAN_OCLIF_CLIENT_HOME=#{lib/"client"}\nCLIENT_HOME="
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin/secman"
+
+    # this variable is used for `update` command
+    # https://github.com/scmn-dev/secman/blob/main/src/commands/update.ts#L48
+    ENV["SM_PROVIDER"] = "brew"
```

Comment:
Nothing you install will see this env variable, so this still does nothing, just like last time.

Reply:
so how I can fix it ?

Reply:
Disable the upgrade command with a setting on build time because it violates: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-tools-that-upgrade-themselves

Reply:
If you need installed executables to see this env variable you need to wrap them in an env script. See [`#write_env_script`](https://rubydoc.brew.sh/Pathname.html#write_env_script-instance_method) or [`#env_script_all_files`](https://rubydoc.brew.sh/Pathname.html#env_script_all_files-instance_method).

Reply:
ok, I removed `ENV["SM_PROVIDER"] = "brew"` line
and secman tests are fixed